core/services/aws/bloodDonation/acceptDonationRequest.ts

```diff
@@ -167,12 +219,20 @@ async function acceptBloodDonationRequest(
 
   const notificationAttributes: DonationNotificationAttributes = {
     userId: seekerId,
-    title: 'Donor Found',
-    status: AcceptDonationStatus.ACCEPTED,
-    body: `${donationPost.requestedBloodGroup} blood found`,
+    title: status === AcceptDonationStatus.ACCEPTED ? 'Donor Found' : 'Donor Ignored',
+    status,
+    body:
+      status === AcceptDonationStatus.ACCEPTED
+        ? `${donationPost.requestedBloodGroup} blood found`
+        : 'unfortunately donor Ignored the request',
```

Comment:
`request was ignored by donor`

Reply:
Updated in fcb443694a4c98a9ce4e50177bafa732c625dd75